### DISCUSSION TITLE
3D Printed Case for egg shaped pumps

### DISCUSSION THREAD

**Discussion by misterpeee** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#265] [Show and tell]
Hello,

My wife has designed a 3D printed case to enable the completed project board to be securely installed directly under the control panel in the "egg shaped" pump units.  Hopefully some others may find this useful too :)

https://www.thingiverse.com/thing:5380738

Thanks

Iain

**Comment by visualapproach** [comment]
Cool, added to readme

**Comment by chunkysteveo** [comment]
Awesome!

**Reply by chunkysteveo** [reply]
Have finally printed this out, amazing work - a perfect print that just fits first time and works. Will update Thingiverse with a "make" shortly. 

FYI - It can be printed without supports if your printer is OK with steep overhangs - it's not going to be seen, so can be F-Ugly!
